src/coreclr/src/pal/src/misc/environ.cpp

```diff
@@ -917,44 +917,51 @@ Return Value
     or nullptr otherwise.
 
 --*/
+static char* match_prefix(const char* name, char** list)
+{
+	if(*name == '\0')
+		return nullptr;
+	
+	for (int i = 0; list[i] != nullptr; ++i)
+	{
+		const char* pch = name;
+		char* p = list[i];
+
+		for (;;) {
```

Comment:
Can you please make the following style changes to match the style in this file / PAL?
* put opening `{` on a separate line
* put space between `if` and the opening `(` in places where you have not done that already